src/screens/ApplicationInfoScreen.tsx

```diff
@@ -1,18 +1,16 @@
 // TODO: there are tons of more interesting methods there!
 import * as Application from 'expo-application'
-import { ScrollView } from 'react-native'
+import { ScrollView, Text } from 'native-base'
 
-import { Text } from '~components'
-import { usePreventGoBack, useTranslation, useTheme } from '~hooks'
+import { usePreventGoBack, useTranslation } from '~hooks'
 
 export const ApplicationInfoScreen = (): JSX.Element => {
   const { i18n } = useTranslation()
-  const { s } = useTheme()
   usePreventGoBack()
 
   return (
-    <ScrollView contentContainerStyle={[s.flex1, s.justifyStart, s.p4]}>
-      <Text>
+    <ScrollView p={4}>
```

Comment:
Before styles was added to content Container and it was because scroll indicator looks better for this specific case. Did you check if this is working fine with this setup?

Reply:
I've checked it and the indicator positions itself correctly. But I found another bug:

Indicator on iOS is visible only when the content of the ScrollView makes the ScrollView scrollable (ok, it is always scrollable but when there is more content than visible on a screen). So that - bug isn't visible now.

But - if there were more content, the indicator's color would not change according to the modal (dark indicator on the white screen and vice versa). I don't see any issue regarding this either on react-native or native-base repository so I am not sure why that is happening.

Nevertheless, we can fix that by changing `indicatorStyle` based on a set color scheme like this:
```tsx
    import { useColorMode } from 'native-base'

    
    const ApplicationInfo: FC = () => {
    const { colorMode } = useColorMode

    return (<ScrollView indicatorStyle={colorMode === 'dark' ? 'white' : 'black'}>
    ...
    </ScrollView>)
```

Reply:
Okay that's good to know. We need to remember about this in the future, but it's not a big issue :P